library/parsedown/LICENSE.txt

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```

Comment:
Move the Parsedown library into a sub-folder of the plugin (e.g. plugin_folder/library/parsedown)